Core/src/Surfaces/LineSurface.cpp

```diff
@@ -204,11 +205,6 @@ Acts::BoundToFreeMatrix Acts::LineSurface::boundToFreeJacobian(
   Vector3 position = freeParams.segment<3>(eFreePos0);
   // The direction
   Vector3 direction = freeParams.segment<3>(eFreeDir0);
-  // Get the sines and cosines directly
-  double cosTheta = std::cos(boundParams[eBoundTheta]);
-  double sinTheta = std::sin(boundParams[eBoundTheta]);
-  double cosPhi = std::cos(boundParams[eBoundPhi]);
-  double sinPhi = std::sin(boundParams[eBoundPhi]);
```

Comment:
@paulgessinger I can only imagine that this one is the reason since we use `LineSurfaces` a lot in vertexing

the direction vector should capture the trig stuff but maybe it is not normalized?